aws-iot-securityprofile/aws-iot-securityprofile.json

```diff
@@ -0,0 +1,345 @@
+          "$ref": "#/definitions/BehaviorCriteria"
+        }
+      },
+      "required": [
+        "Name"
+      ],
+      "additionalProperties": false
+    },
+    "MetricDimension": {
+      "description": "The dimension of a metric.",
+      "type": "object",
+      "properties": {
+        "DimensionName": {
+          "description": "A unique identifier for the dimension.",
+          "type": "string",
+          "pattern": "[a-zA-Z0-9:_-]+",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Operator": {
+          "description": "Defines how the dimensionValues of a dimension are interpreted.",
+          "type": "string",
+          "enum": [
+            "IN",
+            "NOT_IN"
+          ]
+        }
+      },
+      "required": [
+        "DimensionName"
+      ],
+      "additionalProperties": false
+    },
+    "BehaviorCriteria": {
+      "description": "The criteria by which the behavior is determined to be normal.",
+      "type": "object",
+      "properties": {
+        "ComparisonOperator": {
+          "description": "The operator that relates the thing measured (metric) to the criteria (containing a value or statisticalThreshold).",
+          "type": "string",
+          "enum": [
+            "less-than",
+            "less-than-equals",
+            "greater-than",
+            "greater-than-equals",
+            "in-cidr-set",
+            "not-in-cidr-set",
+            "in-port-set",
+            "not-in-port-set"
+          ]
+        },
+        "Value": {
+          "$ref": "#/definitions/MetricValue"
+        },
+        "DurationSeconds": {
+          "type": "integer",
+          "description": "Use this to specify the time duration over which the behavior is evaluated."
+        },
+        "ConsecutiveDatapointsToAlarm": {
+          "description": "If a device is in violation of the behavior for the specified number of consecutive datapoints, an alarm occurs. If not specified, the default is 1.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 10
+        },
+        "ConsecutiveDatapointsToClear": {
+          "description": "If an alarm has occurred and the offending device is no longer in violation of the behavior for the specified number of consecutive datapoints, the alarm is cleared. If not specified, the default is 1.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 10
+        },
+        "StatisticalThreshold": {
+          "$ref": "#/definitions/StatisticalThreshold"
+        }
+      },
+      "additionalProperties": false
+    },
+    "MetricValue": {
+      "description": "The value to be compared with the metric.",
+      "type": "object",
+      "properties": {
+        "Count": {
+          "description": "If the ComparisonOperator calls for a numeric value, use this to specify that (integer) numeric value to be compared with the metric.",
+          "type": "string",
+          "minimum": 0
```

Comment:
Should be `minLength` instead of `minimum`:
https://github.com/aws-cloudformation/cloudformation-cli/issues/414
https://json-schema.org/understanding-json-schema/reference/string.html#length
(caught by running [`cfn validate`](https://github.com/aws-cloudformation/cloudformation-cli/pull/675))